app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -10,6 +11,7 @@ class OtpVerificationController < ApplicationController
     def show
       analytics.multi_factor_auth_enter_otp_visit(**analytics_properties)
 
+      @in_multi_mfa_selection_flow = in_multi_mfa_selection_flow?
```

Comment:
If we defined `in_multi_mfa_selection_flow?` as a helper method, we could have access to it in the templates, and then we wouldn't have to remember to set `@in_multi_mfa_selection_flow` in each controller action

ex `helper :in_multi_mfa_selection_flow?`